SHTSensor.cpp

```diff
@@ -148,7 +148,7 @@ class SHTC1Sensor : public SHTI2cSensor
 public:
     SHTC1Sensor()
         // clock stretching disabled, high precision, T first
-        : SHTI2cSensor(0x70, 0x7866, 15, -45, 175, 65535, 100, 65535)
+        : SHTI2cSensor(0x70, 0x7866, 15, -45, 175, 65535, 100, 65535,0,2)
```

Comment:
same comment as above regarding order: would make more sense to read 
`SHTI2cSensor(0x70, 0x7866, 15, -45, 175, 65535, 0, 100, 65535)` 

(i.e. move 0 up in the argument list; cmd lengths likely not needed)

Also, please match the coding style regarding spaces after commas